app/models/concerns/user_webauthn_methods.rb

```diff
@@ -27,4 +27,12 @@ def webauthn_options_for_get
       user_verification: "discouraged"
     )
   end
+
+  def refresh_webauthn_verification
+    self.webauthn_verification = WebauthnVerification.create(
+      path_token: SecureRandom.base58(16),
```

Comment:
Base 58 chosen because it plays nicely with manually typing in the code and also with direct copy & paste action. 16 characters doesn't impose an undue burden on folks manually typing, but gives a large search space (58^16 ≈ 1.64×10²⁸) that attackers would need to search to find the path token.